### PR TITLE
Update: Change default sorting in directory to most recently updated

### PR DESCRIPTION
**Pull request by daniellevinson**

### CODE REVIEW

File: packages/directory/tests/acceptance/dir-table-test.js [hunk 1]
```diff
@@ -22,36 +22,44 @@ module('Acceptance | dir table', function(hooks) {
...
     );
 
     await click(findAll('th')[3]);
 
     assert.equal(
       currentURL(),
-      '/directory/my-data?sortBy=updatedOn&sortDir=desc',
-      'The sortBy and sortDir are set as query params after sorting'
+      '/directory/my-data?sortDir=asc',
+      'The sortDir is set as query param a after re-sorting by updatedOn'
```
Comment (mattgraham1995) [inline packages/directory/tests/acceptance/dir-table-test.js]: `query param a after` ?

File: packages/directory/tests/acceptance/dir-table-test.js [hunk 2]
```diff
@@ -22,36 +22,44 @@ module('Acceptance | dir table', function(hooks) {
...
     );
 
     assert.deepEqual(
       findAll('.dir-table__cell--lastUpdatedDate').map(elm => elm.innerText.trim()),
       sortedItems.reverse(),
-      'items are sorted by updatedOn desc after sorting'
+      'items are sorted by updatedOn asc after re-sorting by updatedOn'
+    );
+
+    await click(findAll('th')[0]);
```
Comment (mattgraham1995) [inline packages/directory/tests/acceptance/dir-table-test.js]: Is this the same as `click('th')`?

File: packages/directory/tests/acceptance/dir-table-test.js (hunk 1 above)
Comment (daniellevinson) [reply packages/directory/tests/acceptance/dir-table-test.js]: typo :speak_no_evil:

File: packages/directory/tests/acceptance/dir-table-test.js (hunk 2 above)
Comment (daniellevinson) [reply packages/directory/tests/acceptance/dir-table-test.js]: yeah...

**Pull request merged**